dwst/scripts/particles.js

```diff
@@ -186,8 +208,8 @@ function readInstructionArgs(remainder1) {
       return [instructionArgs, remainder];
     }
     if (tmp.charAt(0) !== ',') {
-      const msg = 'syntax error: garbage';
-      throw new InvalidParticles(msg);
+      // expected , but got charAt(0) instead.
```

Comment:
this comment should no longer be necessary?

Reply:
correct.